Source/WebCore/DerivedSources.make

```diff
@@ -2368,6 +2368,18 @@ $(EVENT_NAME_FILES_PATTERNS) : $(WebCore)/dom/make-event-names.py $(EVENT_NAME_J
 
 # --------
 
+# User Agent String overrides
+
+USER_AGENT_STRING_OVERRIDE_JSON := $(WebCore)/page/UserAgentStringOverrides.json
+
+#
+
+all : $(USER_AGENT_STRING_OVERRIDE_JSON)
+$(USER_AGENT_STRING_OVERRIDE_FILES_PATTERNS) : $(WebCore)/page/make-user-agent-string-overrides.py $(USER_AGENT_STRING_OVERRIDE_JSON)
+	$(PYTHON) "$(WebCore)/page/make-user-agent-string-overrides.py" --input $(USER_AGENT_STRING_OVERRIDE_JSON)
```

Comment:
This script needs to be renamed since it's no longer making anything, only validating the json. (Which was a good change.)